packages/server/src/automations/automationUtils.ts

```diff
@@ -122,17 +122,88 @@ export function getError(err: any) {
 }
 
 export function guardAttachment(attachmentObject: any) {
-  if (
-    attachmentObject &&
-    (!("url" in attachmentObject) || !("filename" in attachmentObject))
-  ) {
+  if (!attachmentObject) {
+    return
+  }
+  if (typeof attachmentObject !== "object") {
```

Comment:
Instead of checking the type here, can we not update the signature of the function? 

```ts
export function guardAttachment(attachmentObject: any)
```

I know you'll still have to do that check before the call, but I think it makes sense to correct the typing for this function